platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/TESTS/compliance_attestation/test_a001/main.c

```diff
@@ -20,6 +20,7 @@
 #if !defined(MBED_CONF_RTOS_PRESENT)
 #error [NOT_SUPPORTED] PSA compliance attestation test cases require RTOS to run.
 #else
+
```

Comment:
We should leave this as-is, to avoid diff noise. This isn't necessary to "Add support for compliance_attestation test", so it shouldn't be part of that commit if we even want to make the change at all (I don't think we care to).

Reply:
Removed !